packages/ai-terminal/src/browser/ai-terminal-contribution.ts

```diff
@@ -68,7 +74,8 @@ export class AiTerminalCommandContribution implements CommandContribution, MenuC
                         this.terminalAgent
                     );
                 }
-            }
+            },
+            isEnabled: () => this.shell.currentWidget instanceof TerminalWidget
```

Comment:
The `execute` method checks for `TerminalWidgetImpl` - should the two checks match?